setup.py

```diff
@@ -0,0 +1,44 @@
+from glob import glob
+from os.path import basename, splitext
+
+from setuptools import setup, find_packages
+
+with open("README.rst") as readme_file:
+    readme = readme_file.read()
+
+setup(
+    python_requires=">=3.5",
+    classifiers=[
+        "Development Status :: 2 - Pre-Alpha",
+        "Intended Audience :: Developers",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
+        "Operating System :: Unix",
+        "Operating System :: POSIX",
+        "Natural Language :: English",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Utilities",
+    ],
+    description="Command line tool, that converts Common Cartridge "
+    "courses to Open edX Studio Imports.",
```

Comment:
This doesn't look like it conforms to our style guide; instead maybe:

```py
description=(
    "Command line tool that converts Common Cartridge "
    "courses to Open edX Studio imports"
)
```

Also a nit: `Imports" isn't a proper noun here; it should be lowercase.